packages/frontend-shared/README.md

```diff
@@ -30,7 +30,7 @@ WindiCSS can create an awesome interactive summary showing our usage of utility
 yarn windi
 ```
 
-This will be useful from time to time so that we can audit our usage of these classes and extract repeated patterns into Windi shortcuts or otherwise consolidate them, when it makes sense to do so.
+This will be useful from time to time so that we can audit our usage of these classes and extract repeated patterns into Windi shortcuts or otherwise consolidate them, when it makes sense to do so. 
```

Comment:
Accidental trailing space added

```suggestion
This will be useful from time to time so that we can audit our usage of these classes and extract repeated patterns into Windi shortcuts or otherwise consolidate them, when it makes sense to do so.
```